benchmarks/benchmark_model.py

```diff
@@ -110,21 +111,40 @@ def conversion_dtype(self):
   def prepare_for_experiment(self, dynamo_compilation_opts):
     self.device = self.benchmark_experiment.get_device()
     self.dtype = self.conversion_dtype()
-
     if self.dtype is not None:
       self.module = self.module.to(self.dtype)
       self.example_inputs = cast_to_dtype(self.example_inputs, self.dtype)
 
-    self.module = self.module.to(self.device)
-    self.example_inputs = move_to_device(self.example_inputs, self.device)
-
     if self.benchmark_experiment.test == "eval":
       self._prepare_for_eval()
     elif self.benchmark_experiment.test == "train":
       self._prepare_for_train()
     else:
       raise NotImplementedError
 
+    if self.benchmark_experiment.use_torch_xla2:
+      # for torch_xla2, we export model to FX graph and move weights to JAX device
+      import torch_xla2.export
+      import torch_xla2
+      import jax
+      import jax.numpy as jnp
+      exported = torch.export.export(self.module, self.example_inputs)
```

Comment:
let's add an option to use torch.export

The other (non-export) way is to use `torch_xla2.extract_jax` without exporting first

report both numbers they are both useful to know.

Reply:
Added